src/librustdoc/html/static/main.js

```diff
@@ -408,7 +408,7 @@ function defocusSearchBar() {
                 break;
 
             case "?":
-                if (ev.shiftKey) {
+                if (ev.key || ev.shiftKey) {
```

Comment:
Why not just remove this branch?

Reply:
I've been switching between this and removing the condition for a while. You made the balance change haha.